### DISCUSSION TITLE
Adding group tags in XLIFF

### DISCUSSION THREAD

**Discussion by sudhanshumonga** [maxprograms-com/OpenXLIFF#21] [Ideas]
Currently all the translatable content are put into a list of <trans-unit> tags. For example in HTML the content follows a tree-hierarchical structure, so adding <group> tag may help understand the location of a particular text in the original  file's hierarchy. 
For example: 

HTML: 
{body}
    {div}
        {p}this is para{/p}
        {p}this is another para{/p}
    {/div}
    {div}
        {p}this is para{/p}
        {p}this is another para{/p}
    {/div}
{/body}

XLIFF:
{group id="1"}
    {group id="1.1"}
        {trans-unit}
            {source}this is para{/source}
        {/trans-unit}
        {trans-unit}
            {source}this is another para{/source}
        {/trans-unit}
    {/group}
    {group id="1.2"}
        {trans-unit}
            {source}this is para{/source}
        {/trans-unit}
        {trans-unit}
            {source}this is another para{/source}
        {/trans-unit}
    {/group}
{/group}

currently the XLIFF is like: 
{trans-unit}
    {source}this is para{/source}
{/trans-unit}
{trans-unit}
    {source}this is another para{/source}
{/trans-unit}
{trans-unit}
    {source}this is para{/source}
{/trans-unit}
{trans-unit}
    {source}this is another para{/source}
{/trans-unit}

The former one gives a sense of hierarchy and location in the original file. This thing can be done for DITA and other XML formats as well. 

Let me know about your thoughts on this.

**Comment by rmraya** [comment]
I'm not interested in adding &lt;group> tags. They complicate XLIFF processing.
